optimum/habana/transformers/models/llama/modeling_llama.py

```diff
@@ -23,6 +23,7 @@
     _gaudi_prepare_4d_causal_attention_mask,
 )
 
+import habana_frameworks.torch.core as htcore
```

Comment:
If you rebase to latest then this htcore import is not required, as it is part of PR#65